text/0154-multi-slot-aura.md

```diff
@@ -0,0 +1,84 @@
+This RFC proposes a modification to the AURA round-robin block production mechanism for system parachains (e.g. Polkadot Hub). The proposed change increases the number of consecutive block production slots assigned to each collator from the current single-slot allocation to a configurable value, initially set at four. This modification aims to enhance censorship resistance by mitigating data-withholding attacks.
+
+## Motivation
+
+The Polkadot Relay Chain guarantees the safety of parachain blocks, but it does not provide explicit guarantees for liveness or censorship resistance. With the planned migration of core Relay Chain functionalities—such as Balances, Staking, and Governance—to the Polkadot Hub system parachain in early November 2025, it becomes critical to establish a mechanism for achieving censorship resistance for these parachains without compromising throughput. For example, if governance functionality is migrated to Polkadot-Hub, malicious collators could systematically censor `aye` votes for a Relay Chain runtime upgrade, potentially altering the referendum's outcome. This demonstrates that censorship attacks on a system parachain can have a direct and undesirable impact on the security of the Relay Chain. This proposal addresses such censorship vulnerabilities by modifying the AURA block production mechanism utilized by system parachain collator with minimal honesty assumptions on the collators.  
+
+## Stakeholders
+
+- Collators: Operators responsible for block production on the Polkadot Hub and other system parachains.
+- Users and Applications: Entities that interact with the Polkadot Hub or other system parachains.
+
+## Threat Model
+
+This analysis of censorship resistance for AURA-based parachains operates under the following assumptions:
+
+- **Collator Honesty:** The model assumes the presence of at least one honest collator. We intentionally chose the most relaxed security assumption as collators are not slashable (unlike validators). Note that all system parachains use AURA via the [Aura-Ext](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/pallets/aura-ext) pallet.
+
+- **Backer Honesty:** The backer assigned to a block candidate is assumed to be honest. This is a reasonable assumption given 2/3rd honesty on relay-chain and that backers are assigned randomly by [ELVES](https://eprint.iacr.org/2024/961.pdf). Additionally, we assume that backers responsible for disbursing the withheld block to the victim collators. Pre-PVFs can definitely help in improving the resilience of backers against DoS attacks. Essentially, the pre PVF lets backers check the slot ownership and hence backers can filter out spamming collators at this stage. However, pre-PVFs have not yet been implemented. The stronger on assumption on backer disbursing the block is only needed for efficiency concerns and not essential for censorship resistance itself (i.e. the collator can always reconstruct from the availability layer). 
```

Comment:
Isn't there already some term for Pre-PVF this in the JAM land?  I've called this various different things over the years.